src/services/F2fService.ts

```diff
@@ -238,6 +238,27 @@ export class F2fService {
 		}
 	}
 
+	async updateSessionWithYotiIdAndStatus(sessionId: string, yotiSessionId: string, yotiSessionStatus: string, tableName = this.tableName): Promise<void> {
```

Comment:
Wasn't too keen to bring into scope of this ticket but @richa-misra I think it might make sense to have F2FService, F2FIdentity & F2FSession to contain only relevant DB updates within each file? Or to take the tableName out of the constructor because in documentSelection I initialised F2fService using the PersonIdentity table but then I needed to pass in a diff table name when saving yotiSessionId into Session table as I couldn't use the this.tableName